feat: Deprecated Table in Main (#2344)
Workday/canvas-kit: modules/docs/mdx/10.0-UPGRADE_GUIDE.mdx

	- [useBanner](#useBanner)
- [Deprecations](#deprecations)
  - [InputIconContainer](#input-icon-container)
  - [Table](#table)

>   - [TableRow](#table-row)

Reviewer: You can probably just reference `Table` here and down below mention that all exported members or components are being deprecated

Author: Updated. Could you re-check?

We've deprecated `InputIconContainer` from [Main](#main) because it doesn't handle
bidirectionality or icons at the start of an input. Please consider using [`InputGroup`](https://workday.github.io/canvas-kit/?path=/story/components-inputs-text-input--icons) instead.

### Table

> We've deprecated `Table`, `TableRow` components and all of their exported members. Please consider using [`Table`](https://workday.github.io/canvas-kit/?path=/docs/preview-table--basic) instead as it is more flexible and modern.

Reviewer: ```suggestion
We've deprecated `Table` and `TableRow` as well as all of their exported members. Please consider using [`Table in Preview`](https://workday.github.io/canvas-kit/?path=/docs/preview-table--basic) instead.
```
